src/client/pages/Path/Path.jsx

```diff
@@ -93,8 +93,8 @@ LessonCard.propTypes = {
   type: PropTypes.string.isRequired,
 };*/
 CourseCard.propTypes = {
-  pathId: PropTypes.objectOf(PropTypes.shape).isRequired,
-  courseId: PropTypes.objectOf(PropTypes.shape).isRequired,
+  pathId: PropTypes.string.isRequired,
+  courseId: PropTypes.string.isRequired,
   curriculum: PropTypes.objectOf(PropTypes.shape).isRequired,
```

Comment:
First, there is no shapeOf in proptypes.
But it doesn't really matter because PropTypes.shape(PropTypes.object) makes as little sense as the first version. (Both are just ways to make React not complain but are factually incorrect. objectOf validates an Object with properties of the given parameters type (in our case shape - so it doesn't really make sense) whereas shape() is meant to validate an object of a certain "shape" (certain properties) which means supplying .object as a parameter just tricks the system without any meaningful context.)